Makefile

```diff
@@ -0,0 +1,13 @@
+VENV := venv
+PYLINT := env PYTHONPATH=$(PYTHONPATH) $(VENV)/bin/pylint
+
+lint.py:
```

Comment:
@rahulbot -- do you have opinions about the addition of a `Makefile`? I think it could be a nice way to capture shortcuts, group commands (e.g. `db-setup`), etc.